recipe/meta.yaml

```diff
@@ -12,9 +12,10 @@ source:
     - 0001-Patch-for-downstream.patch
 
 build:
-  number: 1
+  number: 2
   run_exports:
     - {{ pin_subpackage(name, min_pin='x.x', max_pin='x.x') }}
+    - mkl-static
```

Comment:
```suggestion
    - mkl-static  # [win]
```

Reply:
Oops, didn't realize I could just commit from here. Anyway, I made the change. Thanks!